build.gradle

```diff
@@ -4,13 +4,23 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.3'
+        classpath 'com.android.tools.build:gradle:4.0.0'
     }
 }
 
 allprojects {
     repositories {
         google()
+        maven {
```

Comment:
As mentioned above, better to not commit this and to include the gradle.properties without the token.

Reply:
updated the PR with changes